app/views/pages/index.html.erb

```diff
@@ -1,4 +1,22 @@
 <div class="container text-center">
   <h1>Pages#index</h1>
   <p>Find me in app/views/pages/index.html.erb</p>
+  <% if logged_in? %>
+    <h2>Logged-in User:</h2>
+    <hr>
+    <%= current_user.username %><br>
+    <%= current_user.email %>
+    <%= link_to 'Log Out', session_path, method: :delete %>
+    <%= link_to 'Edit Profile', edit_user_path(current_user) %>
+  <% else %>
+    <h2>Welcome, stranger!</h2><br>
```

Comment:
Header elements in HTML are treated as blocks (clear on both sides). You can skip the < br > element in this case. Removing it will also make styling with CSS easier later on.